frontend/src/components/user-management/TeacherUserTable.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { User } from "../../types/UserTypes";
+import { TableRow, Table } from "../common/Table";
+import RemoveUserPopover from "./RemoveUserPopover";
+import { UserTableProps } from "./AdminTab";
+
+const TeacherUserTable = ({ users }: UserTableProps): React.ReactElement => {
```

Comment:
```suggestion
const TeacherUserTable = ({ users }: TeacherUserTableProps): React.ReactElement => {
```
and add 
interface TeacherUserTableProps {
  users: TeacherUser[];
}